sdk/formrecognizer/azure-ai-formrecognizer-perf/src/main/java/com/azure/ai/formrecognizer/perf/core/Utility.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.perf.core;
+
+import com.azure.core.util.logging.ClientLogger;
+import reactor.core.publisher.Flux;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+
+public class Utility {
+    private static final ClientLogger LOGGER = new ClientLogger(Utility.class);
+
+    // using 4K as default buffer size: https://stackoverflow.com/a/237495/1473510
+    private static final int BYTE_BUFFER_CHUNK_SIZE = 4096;
+
+    /**
+     * Creates a Flux of ByteBuffer, with each ByteBuffer wrapping bytes read from the given
+     * InputStream.
+     *
+     * @param inputStream InputStream to back the Flux
+     *
+     * @return Flux of ByteBuffer backed by the InputStream
+     */
+    public static Flux<ByteBuffer> toFluxByteBuffer(InputStream inputStream) {
```

Comment:
This functionality exists in `azure-core`, don't think we need to replicate it here.